data/converters/converters.go

```diff
@@ -0,0 +1,194 @@
+package converters
+
+import (
+	"fmt"
+	"strconv"
+	"time"
+
+	"github.com/grafana/grafana-plugin-sdk-go/data"
+)
+
+// Int64NOOP .....
+var Int64NOOP = data.FieldConverter{
+	OutputFieldType: data.FieldTypeInt64,
+}
+
+// BoolNOOP .....
+var BoolNOOP = data.FieldConverter{
+	OutputFieldType: data.FieldTypeBool,
+}
+
+// Float64NOOP .....
+var Float64NOOP = data.FieldConverter{
+	OutputFieldType: data.FieldTypeFloat64,
+}
+
+// StringNOOP value is already in the proper format
+var StringNOOP = data.FieldConverter{
+	OutputFieldType: data.FieldTypeString,
+}
+
+// AnyToOptionalString any value as a string
+var AnyToOptionalString = data.FieldConverter{
+	OutputFieldType: data.FieldTypeNullableString,
+	Converter: func(v interface{}) (interface{}, error) {
+		if v == nil {
+			return nil, nil
+		}
+		str := fmt.Sprintf("%+v", v) // the +v adds field names
+		return &str, nil
+	},
+}
+
+// AnyToString any value as a string
+var AnyToString = data.FieldConverter{
+	OutputFieldType: data.FieldTypeString,
+	Converter: func(v interface{}) (interface{}, error) {
+		if v == nil {
+			return nil, nil
+		}
+		return fmt.Sprintf("%+v", v), nil // the +v adds field names
```

Comment:
Adding the field names to me seems odd. Why are we doing that?

Reply:
I had used this while pumping an object into a field -- with the names it looked good and without it did not (and has no effect on simple values)

hymmm... what is the behavior when an object implements `toString()`?

Reply:
`toString()` would be nothing as for as fmt is concerned. 

`String()` however is different. When a function has the method in implements fmt's stringer interface https://golang.org/pkg/fmt/#Stringer . `%v	the value in a default format`, will call that method and return its string.

This is actually why String() on dataframe was changed to StringTable, it didn't play well with tests displaying that.